posts/2023-12-08-liberty-certification.adoc

```diff
@@ -0,0 +1,55 @@
+
+In this post, we'll look at how you can prepare yourself for the certification exam.
+
+== Cloud Native Java Developer certification exam
+
+The certification includes multiple-choice questions and two hands-on labs. The hands-on labs are hosted in the cloud so that you can run the exercises in your browser. In the hands-on labs, you are expected to create an application that includes specific features from cloud-native open standards, including MicroProfile and Jakarta EE, and the application should successfully pass integration tests. The certification requires that you gain a minimum of 45 marks out of total of 63 (71%) across both the multiple-choice questions and the hands-on labs to pass. You have 90 minutes to complete the whole exam.
+
+You can find the certification on the link:https://www.ibm.com/training/certification/C9004800[IBM Training portal].
+
+image::img/blog/liberty-certification.png[Screenshot of certification page,width=70%,align="center"]
+
+== Prepare for the certification exam
+
+You can prepare for the certification exam by using the free resources on the link:/[Open Liberty website], including the link:/guides/[cloud-native developer guides] and the link:/docs/[Liberty docs]. Both the guides and docs are relevant whether you are working with Open Liberty or with IBM Liberty (which is built from the Open Liberty project).
+
+=== Cloud-native Java developer guides
+
+The 60 link:/guides/[cloud-native developer guides] help you to get hands-on with the technologies and tools that are available to use with Open Liberty. The guides are a great way for you to refresh your knowledge about a technology or to discover new ones. They range in complexity, from link:/guides/getting-started.html[Getting started with Open Liberty] and link:/guides/rest-intro.html[Creating a RESTful web service] to topics such as link:/guides/microprofile-metrics.html[Providing metrics from a microservice], and link:/guides/social-media-login.html[Authenticating usrs through social media providers].
+
+You can complete a guide in as little as 15 minutes. Additionally, most of the guides are also available for you to "Run in cloud" in a browser, so you don't need to set up any pre-requisites on your local machine. This provides a really easy and efficient way for you to practise using Liberty.
+
+image::img/blog/guides.png[Screenshot of Open Liberty guides,width=70%,align="center"]
+
+As well as the guides, there is a longer link:/guides/liberty-deep-dive.html[Liberty Deep Dive] available. The Liberty Deep Dive combines many of the technologies covered in the shorter guides into a longer tutorial so that you can develop a fully-functional Open Liberty application.
```

Comment:
```suggestion
As well as the guides, there is a longer link:/guides/liberty-deep-dive.html[Liberty Deep Dive] available. The Liberty Deep Dive combines many of the technologies covered in the shorter guides into a longer tutorial so that you can develop a fully-functional cloud-native Java application with Open Liberty using Jakarta EE and MicroProfile APIs.
```